docs/DeveloperGuide.md

```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
    2. **Test case** (Duplicate house): `addSeller n/Carl Lim p/98765432 e/johncarl@example.com type/Hdb street/Clementi Ave 2 blk/311 level/02 unitNo/25 postal/578578 price/999999999` <br>
       **Expected**: No seller is added. Error indicating that the house already existed in the data. <br>
 
+[//]: # (@@author zengzihui)
+### Adding a Buyer
+
+**Prerequisites:**
+- List all persons using the `list` command.
+- There is currently no `person` with the name "James Cook", "Kris Hua". "John Carl 1" exists as a seller in EstateEase.<br>
+
+1. **Add buyer**
+    1. **Test case:** `addBuyer n/James Cook p/98753432 e/jamescook@example.com budget/550000 type/Hdb`<br>
+       **Expected:** A new buyer is added, with name `James Cook`, phone `98753432`, email `jamescook@example.com`, budget `550000`, preferred housing type `Hdb`<br><br>
+
+2. **Invalid format**
+    1. **Test case:** `addBuyer` <br>
+       **Expected:** No buyer is added. Error indicating invalid format with constraints shown. <br><br>
+   
+    2. **Test case:** `addBuyer n/Kris Hua p/98765432 e/krishua@example.com budget/99999900` <br>
+       **Expected:** No buyer is added. Error indicating invalid command format. <br><br>
+
+3. **Invalid (Duplicate Person)**
+    1. **Test case:** `addBuyer n/John Carl 1 p/98765432 e/johncarl1@example.com budget/550000 type/Hdb`
+       **Expected:** No buyer is added. Error indicating that the person already existed in the data. <br><br>
+
+4. **Invalid Value for certain fields**
```

Comment:
Perhaps Invalid Value will do

Reply:
Yes I think the title needs to be changed.